tutorials/ner_tweets/requirements.txt

```diff
@@ -1,10 +1,4 @@
-# NLP
-spacy>=3.1
```

Comment:
This may also be a case where the spacy requirement should be added to `project.yml`?

Reply:
It [has been](https://github.com/rmitsch/spacy-projects/blob/refactor/project-dependent-requirements/tutorials/ner_tweets/project.yml#L63). I don't remember why we bumped the version here - probably ran into some issues testing this with 3.1.

Reply:
I'd just remove it from `requirements.txt` here then.